hspec-core/test/Test/Hspec/Core/RunnerSpec.hs

```diff
@@ -800,7 +800,7 @@ spec = do
       let
         setChatty value args = args { chatty = value }
         withChatty value = hspecCapture [] .  H.modifyArgs (setChatty value) $ do
-          H.it "foo" $ property True
+          H.it "foo" $ once $ property True
```

Comment:
I guess this works as far as Hspec is concerned. I'm still puzzled how this might affect QuickCheck users in general. E.g. a user might experience a slowdown for an existing test suite, without understanding what introduced that slowdown.